CHANGES.rst

```diff
@@ -1,3 +1,45 @@
+1.0
+---
+
+New Features
+^^^^^^^^^^^^
+
+- Implement ``SpectralCoord`` object. [#524] 
+
+- Implement cross-correlation for finding redshift/radial velocity. [#544]
+
+- Improve FITS file identification in default_loaders. [#545]
+
+- Support ``len()`` for ``SpectrumCollection`` objects. [#575]
+
+- Improve JWST data loaders. [#579]
```

Comment:
Could you add #595 to this as well?  I think it will be merged before 1.0 goes out.

Reply:
I think this could be split into two entries:

```suggestion
- Improved 1D JWST loader and allow parsing into an ``SpectrumCollection`` object. [#579]

- Implemented 2D and 3D data loaders for JWST products. [#595] 
```